src/views/dialogs/OnboardingTriggerButton.tsx

```diff
@@ -53,7 +53,9 @@ export const OnboardingTriggerButton = ({
       size={size}
       type={ButtonType.Button}
       state={{
-        isDisabled: disableConnectButton || isAccountViewOnly,
+        isDisabled:
+          disableConnectButton ||
+          (onboardingState === OnboardingState.AccountConnected && isAccountViewOnly),
```

Comment:
logic makes sense. this allows users in WalletConnected state to try to get to AccountConnected.